File: beginner/java/02_sum_two_numbers/Solution.java
/**
 * Challenge 02: Sum Two Numbers
 * TODO: Implement the sum method
 */
public class Solution {
    
    /**
     * Calculate the sum of two integers
     * 
     * @param a First integer
     * @param b Second integer
     * @return The sum of a and b
     */
    public static int sum(int a, int b) {
        return a + b;
        
    }
    
    public static void main(String[] args) {
        // Test your method
        System.out.println("sum(5, 3) = " + sum(5, 3));
        System.out.println("sum(-2, 7) = " + sum(-2, 7));
        System.out.println("sum(0, 0) = " + sum(0, 0));
    }
}

File: beginner/c/02_sum_two_numbers/solution.c
#include <stdio.h>

/**
 * Calculate the sum of two integers
 * 
 * @param a First integer
 * @param b Second integer
 * @return The sum of a and b
 * 
 * TODO: Implement this function
 */
int sum(int a, int b) {
    return a + b;
}

int main() {
    // Test your function
    printf("sum(5, 3) = %d\n", sum(5, 3));
    printf("sum(-2, 7) = %d\n", sum(-2, 7));
    printf("sum(0, 0) = %d\n", sum(0, 0));
    return 0;
}
